packages/liquidator/index.js

```diff
@@ -105,11 +109,19 @@ async function run(
     // instance of Liquidator to preform liquidations.
     const empClient = new ExpiringMultiPartyClient(logger, getAbi("ExpiringMultiParty"), web3, empAddress);
     const gasEstimator = new GasEstimator(logger);
+    const oneInch = new OneInchExchange({
```

Comment:
Does this return `null` or `undefined` or does it throw an error if `oneSplitAddress === undefined`?

Reply:
It'll throw an error